### ISSUE TITLE
How to change the behavior of Ctrl-k in REPL on Windows machine

### ISSUE THREAD

**Issue by BoundaryValueProblems** [julia-vscode/julia-vscode#2278]
Hello. I want to know how to change the keyboard short cut, in particular, the behavior of Ctrl-k in the REPL terminal of VSCode under Windows machines. On macOS, Ctrl-k in the REPL just kill and erase the current command line. But on Windows machine, it doesn't do the same. 
Thanks!

**Comment by davidanthoff** [comment]
If this is in-REPL behavior then this is not a VS Code extension issue, we just surface the standard Julia REPL. I would ask on discourse, I think that is the most likely place to get an answer for this.

**Comment by BoundaryValueProblems** [comment]
@davidanthoff ; I am not sure whether this is purely the standard Julia REPL issue. I tried REPL outside of the VSCode, and found out that Ctrl-k erase the current line as expected. I tried this both on git-bash terminal and powershell. But the Ctrl-k of the REPL within the VSCode behave differently. That's why I asked this question in the first place.

**Comment by davidanthoff** [comment]
So if I start Julia from the start menu on Windows, Ctrl+K does not delete anything, same if I start PowerShell (either in Windows Terminal or from the legacy console).

**Comment by BoundaryValueProblems** [comment]
Well, I just checked the following: running Julia on plain cmd and running it via start menu on Windows. Both cases, when I type, say, `julia> pwd()` (without hitting the return key), followed by ctrl-a (which moves the cursor to the head of this command), followd by ctrl-k erases the command `pwd()`. I wanted to have this as in the REPL on VSCode, but somehow, it behaves differently. By the way, on the plain cmd, if I do not invoke Julia/REPL, and type plain DOS or Windows commands, ctrl-k does not erase those commands. So, Julia REPL does something here. By the way, I use OhMyREPL on all of my systems, i.e., macOS, Windows, and Linux. I also use it under Julia REPL session via VSCode. The shortcut keys work as expected except the combination of Windows + VSCode. All the other combinations work.

**Comment by davidanthoff** [comment]
Ah, actually, I know what the problem is: Ctrl+K is the start of some combined keyboard commands in VS Code. You can see that in the status bar is says this

![image](https://user-images.githubusercontent.com/1036561/125174068-94e00800-e177-11eb-9fab-155a30515173.png)

So my best guess is that if you were to find all keyboard shortcuts in VS Code that have a shortcut that starts with Ctrl+k and then remap them, maybe Ctrl+k would then be considered "free" by VS Code and actually sent to the terminal...

**Comment by BoundaryValueProblems** [comment]
Thanks a lot! I defined the keybinding on VSCode, in particular, I assigned `Delete All Right` function to `ctrl-k`. On macOS, it has been defined from the beginning. But on Windows, no keybinding was set for `Delete All Right`. Now, I could do this in the same way throughout the systems! However, when I assigned that function to `ctrl-k` on Windows, it warned that there are other keys starting with `ctrl-k`, which you mentioned, but since I may not use other commands with `ctrl-k` followed by other keys, so at this point, I'm quite satisfied. Thanks!
